### ISSUE TITLE
FROM php:7.1-fpm = ERROR: failed to create new listening socket: socket(): Address family not supported by protocol (97)

### ISSUE THREAD

**Issue by Aielen** [docker-library/php#987]
Hi, I think, I found a bug in the Docker engine. 

I have following Dockerfile:
```
FROM php:7.1-fpm
```

Very simple container. But when I am starting this container I got following error:
```
ERROR: failed to create new listening socket: socket(): Address family not supported by protocol (97)
ERROR: failed to create new listening socket: socket(): Address family not supported by protocol (97)
ERROR: FPM initialization failed
ERROR: FPM initialization failed
```

I am using Windows 10 Enterprise (latest version).  Today I also updated my Docker to the latest version (19.03.8). Before that Docker engine update, everything worked fine. 

I decided to revert my old Docker version.

Fortunately, I have on my HDD in the "Downloads" folder Docker installer in version:
```
$ docker --version
Docker version 19.03.5, build 633a0ea
```

After Docker version downgrade, php71 container worked again. 

I think that Docker developers have broken something in the newest Docker version :(

**Comment by wglambert** [comment]
This is a recent issue with that same error message
https://github.com/docker/for-win/issues/6206
>You'll get "2020/04/02 16:37:51 [emerg] 1#1: socket() [::]:80 failed (97: Address family not supported by protocol)" because ipv6 seems unsupported inside the linux container.

Seems like a Docker for Windows issue, not necessarily something we can fix as image maintainers

**Comment by Aielen** [comment]
Thanks for your answer. So it looks like we need to wait for a next Docker release, in which, hopefully, Docker Team will fix this bug. 

If somebody also got this problem - here is a workaround; downgrade Docker version and everything will work fine (to version `19.03.5`)

**Comment by Aielen** [comment]
I updated Docker for Windows again, and they fixed it.
```
$ docker --version
Docker version 19.03.8, build afacb8b
```

Still version 19.03.8, but different build hash.

**Comment by diosby** [comment]
Turn ipv6 on on PC. 
I use Ubuntu 19.04. Tried 19.03.8, 19.03.3, 19.03.12 and I got the same problem.
After I turn ipv6 on and the problem was gone.
Command `ip a | grep inet6` must shows addresses.

**Comment by dvdn** [comment]
For the record,
yes it's IPv6 related, as this docker image `php:7.1-fpm` doesn't work on a host with IPv6 disabled
(for example on Linux in your boot settings in grub file `GRUB_CMDLINE_LINUX_DEFAULT="ipv6.disable=1 ...`).

It has been fixed since the `php:7.2-fpm` image : https://bugs.php.net/bug.php?id=74166

And it works trying IPv6 then on IPv4 :
```
root@9289da4bbd42:/srv/app# php-fpm
NOTICE: Failed implicitly binding to ::, retrying with 0.0.0.0
NOTICE: fpm is running, pid 42
NOTICE: ready to handle connections
...

```
